inductiva/coastal/bathymetry.py

```diff
@@ -275,6 +275,12 @@ def y_uniques(self, sort: bool = False) -> np.ndarray:
             y_uniques = np.sort(y_uniques)
         return y_uniques
 
+    @optional_deps.needs_coastal_extra_deps
```

Comment:
Now that I am thinking, I think we do not need all of these optional_deps, since the numpy is actually in the default package. Only the plots should require this. Let's remove them in another PR, not urgent.